src/domain/in-app-notification-reader/dto/in.app.notification.callout.published.ts

```diff
@@ -12,9 +12,15 @@ import { InAppNotificationBase } from './in.app.notification.base';
   implements: () => InAppNotification,
 })
 export abstract class InAppNotificationCalloutPublished extends InAppNotificationBase() {
-  type!: NotificationEventType.COLLABORATION_CALLOUT_PUBLISHED;
-  payload!: InAppNotificationCalloutPublishedPayload;
+  type = NotificationEventType.COLLABORATION_CALLOUT_PUBLISHED;
```

Comment:
this has to remain - we narrowing the type of `NotificationEventType` to `NotificationEventType.COLLABORATION_CALLOUT_PUBLISHED`